src/preprint_bot/preprint_bot_summarization.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+"""
+preprint_bot_summarization.py
+
+Fetches arXiv papers, saves text/metadata, and summarizes abstracts using transformer models.
+"""
+
+import argparse
+from pathlib import Path
+from urllib.parse import quote
+import feedparser
+import re
+import csv
+from summarization_script import summarize_with_transformer  # Transformer-based summarization function
+
+# ------------------------
+# Folders for input/output
+# ------------------------
+INPUT_FOLDER = Path("input_folder")     # Folder to store raw paper text files
+OUTPUT_FOLDER = Path("summaries")       # Folder to store summarized outputs
+
+def create_folders():
+    """Create input and output folders if they don't already exist."""
+    INPUT_FOLDER.mkdir(parents=True, exist_ok=True)
+    OUTPUT_FOLDER.mkdir(parents=True, exist_ok=True)
```

Comment:
Make the folders input arguments. It can take defaults if you like; i.e.:
```suggestion
def create_folders(input_folder="input_folder", output_folder="summaries"):
    Path(input_folder).mkdir(parents=True, exist_ok=True)
    Path(output_folder).mkdir(parents=True, exist_ok=True)
```